### PR TITLE
[all] Cleanup unformatted files

### PR DESCRIPTION
**Pull request by Rexios80**
The format command always changes these even though I haven't touched them

https://github.com/flutter/flutter/issues/152651

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I [linked to at least one issue that this PR fixes] in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style], or this PR is [exempt from CHANGELOG changes].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/QueueUtils.h [hunk 1]
```diff
@@ -7,7 +7,7 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /// Queue-specific context data to be associated with the capture session queue.
-extern const char* FLTCaptureSessionQueueSpecific;
+extern const char *FLTCaptureSessionQueueSpecific;
```
Comment (stuartmorgan) [inline packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/QueueUtils.h:10]: This probably depends on the exact version of clang-format, due to changes in the logic for detecting whether a file is C++ or Obj-C.

File: packages/camera/camera_windows/windows/capture_controller.cpp [hunk 2]
```diff
@@ -39,7 +39,7 @@ CaptureControllerImpl::CaptureControllerImpl(
     : capture_controller_listener_(listener),
       media_settings_(
           PlatformMediaSettings(PlatformResolutionPreset::kMax, true)),
-      CaptureController(){};
+      CaptureController() {};
```
Comment (stuartmorgan) [inline packages/camera/camera_windows/windows/capture_controller.cpp:42]: All of the rest of these are minor differences in how clang-format handles this code, but the easy solution to these is to fix them all to not have the stray `;` that shouldn't be there and is causing it to have non-trivial format analysis. Every one of these should have stable formatting across versions once the incorrect `;` is gone.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

**Comment by stuartmorgan** [comment]
(FYI, the CI runs the same format command that you are running locally, and fails on diffs, so there are never files checked in that `format` touches but that aren't formatted.)

File: packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/QueueUtils.h (hunk 1 above)
Comment (Rexios80) [reply packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/QueueUtils.h:10]: I think I'm using the latest clang-format version. What version is CI using? Or has this file just not been touched in a really long time?

File: packages/camera/camera_windows/windows/capture_controller.cpp (hunk 2 above)
Comment (Rexios80) [reply packages/camera/camera_windows/windows/capture_controller.cpp:42]: I removed the `;` and the formatter still adds the space. Is that the expected behavior?

**Comment by Rexios80** [comment]
> (FYI, the CI runs the same format command that you are running locally, and fails on diffs, so there are never files checked in that format touches but that aren't formatted.)

I figured that was the case. Is CI using an old clang-format version? Or does it only check files that changed in the PR.

**Comment by stuartmorgan** [comment]
> Is CI using an old clang-format version?

It's using a specific git revision specified in the CI config; I don't know the exact clang version it corresponds to.

> Or does it only check files that changed in the PR.

Like almost all the repo commands, in CI it runs on any touched packages, or on all packages for repo-level changes (such as the daily Flutter roll).

File: packages/camera/camera_windows/windows/capture_controller.cpp (hunk 2 above)
Comment (stuartmorgan) [reply packages/camera/camera_windows/windows/capture_controller.cpp:42]: Yes, the space is correct.

**Comment by Rexios80** [comment]
> It's using a specific git revision specified in the CI config; I don't know the exact clang version it corresponds to.

What is the reason for this? Should we update it?

Should we update the documentation on how to setup the correct version of clang-format? I've accidentally checked in these formatting changes a few times and goofed up PRs.

**Comment by stuartmorgan** [comment]
> > It's using a specific git revision specified in the CI config; I don't know the exact clang version it corresponds to.
> 
> What is the reason for this?

The reason for using specific builds in CI is to ensure repeatable behavior in CI.

> Should we update it?

If there's a newer CIPD package already available, I'd be happy to review a PR to update it. It was current as of about a year ago, and there hasn't been a need to update it.

> Should we update the documentation on how to setup the correct version of clang-format? I've accidentally checked in these formatting changes a few times and goofed up PRs.

It's not-trivial to set up an identical version locally, and this doesn't come up all that often. There's an issue tracking doing this IIRC, but it hasn't been a priority. I'm happy to review a PR for that as well.

**Comment by Rexios80** [comment]
We should definitely do _something_ about this. It's not a great developer experience (especially for new contributors) to have untouched files change by running the format command.

**Comment by Rexios80** [comment]
@stuartmorgan See https://github.com/flutter/packages/pull/7271

**Comment by stuartmorgan** [comment]
> We should definitely do _something_ about this. It's not a great developer experience (especially for new contributors) to have untouched files change by running the format command.

If clang is updated to a newer version that what all the contributors who don't currently have that problem are using, then they will start having that problem. There's no solution that will work for everyone unless the repo tooling manages a `clang-format` version itself (which is a lot of extra complexity).

**Pull request closed without merging**